src/templates/base.html

```diff
@@ -88,7 +88,7 @@
   <hr>
   <div class="container">
     <footer>
-      © CubingUSA 2017.
+      © CubingUSA {% print(c.currentDate.year) %}.
```

Comment:
Jinja lets you use

{{ c.currentDate.year }}

as a shorthand instead of print

Reply:
Very cool!